streamer/src/nonblocking/quic.rs

```diff
@@ -301,6 +301,7 @@ async fn run_server(
     let stream_load_ema = Arc::new(StakedStreamLoadEMA::new(
         stats.clone(),
         max_unstaked_connections,
+        max_staked_connections,
```

Comment:
In the rest of the code, staked_connections come before the unstaked ones in the args